pages.pt_BR/android/screencap.md

```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Tira uma scrennshot do display mobile.
+> Este comando apenas pode ser usado atraves de `adb shell`.
+> Mais informacoes: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Tira uma screenshot:
+
+`adb shell screencap {{path/to/file}}`
```

Comment:
Oh, this is an error. if this page is on `android` platform. Please apply this fix to the screencap pages.
```suggestion
`screencap {{path/to/file}}`
```

Reply:
Ok, I'll fix it!